CMakeLists.txt

```diff
@@ -89,8 +89,9 @@ set(COSTOVERRIDE_SVE 10)
 set(COSTOVERRIDE_SVENOFMA 10)
 set(COSTOVERRIDE_RVVM1 10)
 set(COSTOVERRIDE_RVVM1NOFMA 10)
-set(COSTOVERRIDE_RVVM2 10)
-set(COSTOVERRIDE_RVVM2NOFMA 10)
+set(COSTOVERRIDE_RVVM2 20)
+set(COSTOVERRIDE_RVVM2NOFMA 20
+)
```

Comment:
nit: can we please close parenthesis on same line?

Reply:
Done.